### PR TITLE
Create the ROADMAP for 2023 Q1

### PR DESCRIPTION
**Pull request by brunoocasali**
I don't think it is worth mentioning more about the tiers now. Since I'm working on the article, I can link it after the first submission.

Remember the Q2 starts in April 1st.

### CODE REVIEW

**Review by alallema — DISMISSED** [review]
Perfect for me! 🥯

File: roadmap/2023.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Roadmap 2023
+
+This is the roadmap planned by the Integration team at Meilisearch for 2023.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2023: the Main Lines
+
+#### Objectives
```
Comment (curquiza) [inline roadmap/2023.md:9]: I would add "Focus on the impact: spend time updating SDKs that are mostly used, and work with the community for the other ones", or something similar

**Review by curquiza — COMMENTED** [review]
Nice work!

I would add a reminder regarding the OKR, and also justify with project is linked to which OKR.

For example

```
[ ] Add the usage support for the [autocomplete.js](https://github.com/algolia/autocomplete) (O1KR2)
```
I don't remember which OKR is what, this is just an example 😇

File: roadmap/2023.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Q1
+
+We're entering a new era in 2023, and now we want to achieve some goals we were unable to last year. It is time to increase our DevOps CI's reliability and take the opportunity to be present in two of the biggest cloud providers: Azure and Alibaba. Together they control 26% of the market share. This could be a good advertisement for the company and bring more users.
+
+We are also entering a phase where new projects, not just integrations, will be requested by the team, so we must know where to spend our precious maintenance time. That's why we're going to introduce SDK tiers.
+
+
+Processes:
+- [ ] Introduce the versioning policy for the post-v1 era. (O1K1)
```
Comment (curquiza) [inline roadmap/2023.md:26]: Where are described O1K1?

**Review by curquiza — CHANGES_REQUESTED** [review]

File: roadmap/2023.md (hunk 2 above)
Comment (brunoocasali) [reply roadmap/2023.md:26]: I tought you were asking that information just for quickly recognition. I didn't think you would like to have the OKR written here. 
I don't think if I share publically the OKRs they will bring much value for the community.

Comment (brunoocasali) [reply roadmap/2023.md:37]: But in any case, I added them in the top of the file, check it out 😄

File: roadmap/2023.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- [ ] Add the usage support for the [autocomplete.js](https://github.com/algolia/autocomplete), this project will be finished by the end of the Q1. (O2K2)
+- [ ] Implement **packer.io** to build Meilisearch images for the cloud providers, this project will be started during this quarter and finished in the Q2 (O2K1, O2K2)
+
+Maintenance:
+- [x] Add Meilisearch engine version v1.0.0 support for the integrations and SDKs. (O1K2)
+- [ ] Add Meilisearch engine version v0.30 support for Swift. (O1K2)
+- [ ] Add Meilisearch engine version v1.0.0 support for Swift. (O1K2)
+- [x] Add Meilisearch engine version v0.29 support for Java. (O1K2)
+- [ ] Add Meilisearch engine version v0.30 support for Java. (O1K2)
+- [ ] Add Meilisearch engine version v1.0.0 support for Java. (O1K2)
```
Comment (alallema) [inline roadmap/2023.md:50]:
```suggestion
- [x] Add Meilisearch engine version v1.0.0 support for Java. (O1K2)
```
:zany_face:

**Review by alallema — APPROVED** [review]
🥯 🥯 🥯

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by curquiza** [comment]
@brunoocasali I discussed with Quentin and it's completely fine to share the OKR publically

**Pull request merged**